oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -738,6 +804,13 @@ Builder setEnvironmentProvider(EnvironmentProvider environmentProvider) {
       return this;
     }
 
+    /** Sets the optional service account impersonation options. */
```

Comment:
Should be public. Can take in a map if you don't want to expose ServiceAccountImpersonationOptions